Sources/MatrixClient/API/Events/RoomEvent.swift

```diff
@@ -8,9 +8,9 @@ public protocol MatrixEvent: Codable {
     static var type: String { get }
 
     // var content: Content { get }
-    var eventID: String { get }
-    var sender: String { get }
-    var date: Date { get }
+    var eventID: String? { get }
+    var sender: String? { get }
+    var date: Date? { get }
```

Comment:
Yeah, I’m still pretty much against making these optional. If I have a sync response I should be able to rely on these fields being there and not have to unwrap them for every single event. 😕

Reply:
My problem is that then we have to have every room event 2 times